.circleci/config.yml

```diff
@@ -33,7 +33,6 @@ jobs:
             aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/sandbox dev_env.sh
             source dev_env.sh
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
-            pip install .[dev]
```

Comment:
Is this removed intentionally? If it passes without it then it doesn't matter

Reply:
Yes, installing this into the tap-teser venv adds a whole bunch of headaches for dependencies and is actually not necessary to the tests.